pkg/kubelet/status/status_manager_test.go

```diff
@@ -622,6 +622,64 @@ func TestTerminatePod(t *testing.T) {
 	assert.Equal(t, newStatus.Message, firstStatus.Message)
 }
 
+func TestTerminatePodWaiting(t *testing.T) {
```

Comment:
Basically copied from test above, but changed (init)container 3 to waiting status